src/database/sqlite/sqlite_aclk_alert.c

```diff
@@ -219,245 +213,438 @@ static inline char *sqlite3_text_strdupz_empty(sqlite3_stmt *res, int iCol) {
     return strdupz(ret);
 }
 
+#define SQL_UPDATE_ALERT_VERSION                                                                                       \
+    "INSERT INTO alert_version (health_log_id, unique_id, status, version, date_submitted)"                            \
+    " VALUES (@health_log_id, @unique_id, @status, @version, @date_submitted)"                                         \
+    " ON CONFLICT(health_log_id) DO UPDATE SET status = excluded.status, version = excluded.version, "                 \
+    " unique_id=excluded.unique_id, date_submitted=excluded.date_submitted"
+
+//
+// Store a new alert transition along with the version after sending to the cloud
+//   - Update an existing alert with the updated version, status, transition and date submitted
+//
+static void sql_update_alert_version(int64_t health_log_id, int64_t unique_id, RRDCALC_STATUS status, uint64_t version)
+{
+    static __thread sqlite3_stmt *res = NULL;
+
+    if (!PREPARE_COMPILED_STATEMENT(db_meta, SQL_UPDATE_ALERT_VERSION, &res))
+        return;
+
+    int param = 0;
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, health_log_id));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, unique_id));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int(res, ++param, status));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, version));
+    SQLITE_BIND_FAIL(done, sqlite3_bind_int64(res, ++param, now_realtime_sec()));
```

Comment:
Above we are using `UNIXEPOCH()` to set a date and here we are using `now_realtime_sec()` . Why are we not using the same function on both?

Reply:
True, I will change it -- no need to bind the extra param